php/laravel/config.yaml

```diff
@@ -17,3 +17,6 @@ before_command:
   - find /usr/src/app -type f -exec chmod 644 {} \;
   - find /usr/src/app -type d -exec chmod 755 {} \;
   - chmod -R ug+rwx storage /usr/src/app/bootstrap/cache
+
+  - php artisan config:cache
+  - php artisan route:cache
```

Comment:
is the warm-up not enough ?
https://github.com/the-benchmarker/web-frameworks/blob/b99ea681988502f59508df4062aaf48372e81dbd/.tasks/collect.rake#L37

Reply:
No, warmup does not cache either the config or routes. This has to be done manually upon deployment.

Reply:
ok, so forgot what I said about `artisan` binary